src/pages/__tests__/ContactUsView.test.js

```diff
@@ -1,13 +1,58 @@
 import React from 'react';
-import { render } from 'test-utils';
+import { render, fireEvent } from 'test-utils';
 
 import ContactUsView from '../ContactUsView';
 
 const PAGE_HEADER = 'Contact Us';
+const SEND = /send/i;
+const EXPECTED_OUTPUT = 'mailto:oneleifdev@gmail.com?subject=this%20is%20a%20test&body=this%20is%20a%20test';
+const TEST_INPUT = 'this is a test';
+const MESSAGE_TEXTAREA = 'Message-textarea';
+const SUBJECT_INPUT = 'Subject-input';
 
 describe('Contact Us View Tests', () => {
+  function setUp() {
+    const { queryByText, queryByLabelText } = render(<ContactUsView />);
+    return { queryByText, queryByLabelText };
+  }
```

Comment:
Why not return the render instead like so? 

``` javascript
return render(<ContactUsView />);
```
That way I can comment about how lazy you are!

Reply:
In doing that can i just pull out what's needed? I'm still learning too man! lol

Reply:
Yup, you should be able to, but I think you should leave it as is because the pattern is clearer at first glance. I like how we've a setup that returns exactly what we need and nothing more.

Reply:
Damn i just updated it lol, should i revert?

Reply:
Hahaha! No need to revert; it's fine.